lib/src/test/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueResourceTest.kt

```diff
@@ -10,13 +10,15 @@ import org.junit.jupiter.api.Test
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
 import org.springframework.boot.test.mock.mockito.MockBean
 import org.springframework.test.web.servlet.MockMvc
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
 
-@WebMvcTest
+@WebMvcTest(HmppsQueueResource::class)
+@AutoConfigureMockMvc(addFilters = false)
```

Comment:
why do you need this?

Reply:
To turn off the security filter.  WebMvcTest is fine with security when you are testing an real application but for testing a library I would need to build a real application to add security in... oh yeah I did that in the `test-app` module :smile: 

So essentially this class is a "unit" test for the resource.